### ISSUE TITLE
Missing storyboard templates

### ISSUE THREAD

**Issue by taykay08** [SwiftGen/SwiftGen#532]
I did a fresh install of SwiftGen v6.0 using Homebrew today, and no bundled storyboard templates were available/visible in the list of templates.

`swiftgen templates list` results:

```
colors:
  custom:
  bundled:
   - literals-swift4
   - swift3
   - literals-swift3
   - swift4
fonts:
  custom:
  bundled:
   - swift3
   - swift4
ib:
  custom:
  bundled:
   - scenes-swift4
   - segues-swift4
   - segues-swift3
   - scenes-swift3
json:
  custom:
  bundled:
   - runtime-swift4
   - inline-swift4
   - inline-swift3
   - runtime-swift3
plist:
  custom:
  bundled:
   - runtime-swift4
   - inline-swift4
   - inline-swift3
   - runtime-swift3
strings:
  custom:
  bundled:
   - flat-swift4
   - structured-swift4
   - flat-swift3
   - structured-swift3
xcassets:
  custom:
  bundled:
   - swift3
   - swift4
yaml:
  custom:
  bundled:
   - inline-swift4
   - inline-swift3
storyboards:
  custom:
  bundled:
---
You can add custom templates in /Users/.../Library/Application Support/SwiftGen/templates.
You can also specify templates by path using `templatePath` instead of `templateName`.
For more information, see the documentation on GitHub.
```

**Comment by AliSoftware** [comment]
Hi,

Thanks for the issue

This is a sticker of #525 and has been solved already 😉

I recommend you go read #525 where we give some pointers to the migration guide btw, as the `storyboards` as been deprecated in favor of `ib` (see details in the MG)

**Comment by djbe** [comment]
Hi @taykay08,

This is a duplicate of #525, it's been fixed in #530. We'll release a bugfix version in a day or two.

**Comment by djbe** [comment]
Gah, sniped!
